action/protocol/poll/util.go

```diff
@@ -169,7 +169,11 @@ func setCandidates(
 	}
 	loadCandidatesLegacy := featureCtx.LoadCandidatesLegacy(height)
 	for _, candidate := range candidates {
-		delegate, err := accountutil.LoadOrCreateAccount(sm, candidate.Address)
+		addr, err := address.FromString(candidate.Address)
```

Comment:
poll is no longer used, so as long as it can pass fullsync test, it will be fine